### ISSUE TITLE
Document source of .gitignores

### ISSUE THREAD

**Issue by bbodenmiller** [joeblau/gitignore.io#112]
It'd be great if https://github.com/github/gitignore was identified as one of the sources of .gitigore files.

**Comment by joeblau** [comment]
Where would you like to see that identified?

**Comment by bbodenmiller** [comment]
Just somewhere in the README and on the docs section of the website. Perhaps in a section about how to contribute to the .gitignore templates.

**Comment by joeblau** [comment]
That has been in [CONTRIBUTING.md](https://github.com/joeblau/gitignore.io/blob/master/CONTRIBUTING.md) since September of 2014 which is the same time I added GitHub's ignore as a submodule.  This is also inline with [GitHub's Contributing Guidelines](https://github.com/blog/1184-contributing-guidelines).  That being said, maybe I should add it somewhere else because a lot of people don't immediately seem to realize that there is a submodule.

**Comment by bbodenmiller** [comment]
Glad to see it in CONTRIBUTING.md. My main thing was that I skimmed the README and couldn't find the source of the gitignore's nor where they were located in the repo. Additionally CONTRIBUTING.md says to PR to files to `/data/gitignore/Custom`. I think the community would be better off it they supported them to https://github.com/github/gitignore and as a backup submitted them to your repo if gitignore does not want to accept.

**Comment by joeblau** [comment]
> I think the community would be better off it they submitted them to https://github.com/github/gitignore and as a backup submitted them to your repo if gitignore does not want to accept.

That is how it works.

**Comment by bbodenmiller** [comment]
I think a bullet under `Adding/Updating Templates` in CONTRIBUTING would make that a bit more clear.

**Comment by joeblau** [comment]
:+1: go ahead and add what you think.  I'll merge it.

**Comment by joeblau** [comment]
So @bbodenmiller I'm actually going to close this because I just went back though some history and remembered why I created his project.  The whole reason this project was created was because GitHub wasn't actually merging any PRs for _months_.  I actually went though manually, commented on and merged a lot of PR's into my library at the time.  So while GitHub has a good source, I'm still not going to recommend that people contribute there first as the original vision of this project is to become the authoritative source (which is also stated in the CONTRIBUTING.md).
